repos/system_upgrade/el7toel8/actors/networkmanagerreadconfig/libraries/library.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from six.moves.configparser import ConfigParser, ParsingError
+
+from leapp.libraries.stdlib import CalledProcessError, run, api
+
+
+def _is_file_readable(path):
+    ''' Verify if file exists and is readable '''
+    return os.path.isfile(path) and os.access(path, os.R_OK)
+
+
+def read_nm_config(file_path=None):
+    # import pdb
+    # pdb.set_trace()
+    # file_path = os.path.join(os.getcwd(), file_path)
+    if file_path:
+        if _is_file_readable(file_path):
```

Comment:
What's the point of checking the existence of the file? It's better to use the existing Python's functionality - try/except.